packages/flutter/lib/src/material/drawer.dart

```diff
@@ -190,6 +192,19 @@ class DrawerController extends StatefulWidget {
   /// Optional callback that is called when a [Drawer] is opened or closed.
   final DrawerCallback drawerCallback;
 
+  /// {@template flutter.material.drawer.dragStartBehavior}
+  /// Determines the way that drag start behavior is handled.
+  ///
+  /// If set to [DragStartBehavior.start], drawer opening drag behavior will
```

Comment:
I am not sure I fully understand this paragraph. Questions that a user may have when reading this paragraph, that should be answered (either directly here or by adding a see also section with helpful links): Whats drawer opening drag behavior? (Does this also apply to closing the drawer?) And whats the difference between the detection of a drag gesture and the initial down event?